src/main/java/com/softserve/skillscope/mapper/TalentMapper.java

```diff
@@ -0,0 +1,9 @@
+package com.softserve.skillscope.mapper;
+
+import com.softserve.skillscope.talent.model.entity.Talent;
+import com.softserve.skillscope.talent.model.entity.TalentFlashcard;
+import org.mapstruct.Mapper;
+@Mapper
+public interface TalentMapper {
+    TalentFlashcard toTalentFlashcard(Talent talent);
```

Comment:
If you use the mapstruct processor, after removing implementation class, change the signature of this interface to method or _**similar**_ to this.
```
    default TalentFlashcard toTalentFlashcard(Talent talent) {
      
        return new TalentFlashcard(talent.getId(),
                talent.getTalentInfo().getImage(),
                talent.getName(),
                talent.getSurname(),
                talent.getTalentInfo().getLocation());

    }